ads/adpon.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adpon(global, data) {
+  validateData(data, ['fid'], ['debugScript']);
+
+  global._adpon = {fid: data['fid']};
+
+  writeScript(global, data['debugScript'] || 'https://ad.adpon.jp/amp.js', () => {
+    global.context.renderStart();
```

Comment:
this calls renderStart no matter if the ad returned or not, which is not we want here.

the ideal place is to call it in your https://ad.adpon.jp/amp.js when there is ad returned. in case of no-fill, `context.noContentAvailable()` should be called instead.

Reply:
Thank you for your review.
I can not implement renderStart in our current code.
I want to avoid implementing renderStart, is it allowed?

Reply:
that's fine. feel free to revert.